deploykit/__init__.py

```diff
@@ -471,6 +474,7 @@ def run(
         # FIXME we assume bash to be present here? Should be documented...
         ssh_cmd = (
             ["ssh", ssh_target]
+            + self.extra_ssh_opts
```

Comment:
Would you not rather add this after the normal arguments (ssh_opts) to be able to overwrite existing ones?

Reply:
Good idea, done!